scene/gui/tree.cpp

```diff
@@ -6026,6 +6052,11 @@ Tree::Tree() {
 	range_click_timer->connect("timeout", callable_mp(this, &Tree::_range_click_timeout));
 	add_child(range_click_timer, false, INTERNAL_MODE_FRONT);
 
+	dropping_unfold_timer = memnew(Timer);
+	dropping_unfold_timer->set_wait_time(theme_cache.dragging_unfold_wait_msec * 0.001);
```

Comment:
You are passing the time to `start()`, so setting it here is unnecessary.
Also you should enable `one_shot` for the Timer (it's off by default).

Reply:
Setting it here is necessary or else changing the editor setting would require a restart.